### ISSUE TITLE
Problem

### ISSUE THREAD

**Issue by rastaman111** [yoheimuta/RxMusicPlayer#39]
I can use any audio files in the application.
When the file has a complete metadata, then in MPMediaItemPropertyTitle, MPMediaItemPropertyArtist, MPMediaItemPropertyArtwork, and if something is missing then an empty line in this place, how can I substitute my values if empty?

found that this function is updating data private func updateNowPlayingInfo () -> Observable <()> {

**Comment by yoheimuta** [comment]
> how can I substitute my values if empty?

@rastaman111 Not really. You can set an item to your complete metadata, but it also skips downloading metadata.
Your request needs a small change on the library. What do you think?

**Comment by rastaman111** [comment]
> > how can I substitute my values if empty?
> 
> 
> 
> @rastaman111 Not really. You can set an item to your complete metadata, but it also skips downloading metadata.
> 
> Your request needs a small change on the library. What do you think?

I think it is necessary to do it by default so that when there is no metadata, then by default substitute the values ​​that are on the player's screen, these are: title, artist and picture

![image](https://user-images.githubusercontent.com/15727518/135704729-8c1f6570-60c5-4105-8977-aa16d64ce434.png)
![image](https://user-images.githubusercontent.com/15727518/135704732-e7d31d38-cbee-417e-a392-ccafdd16abe4.png)
